proto/gradle.proto

```diff
@@ -8,6 +8,7 @@ package gradle;
 
 service Gradle {
   rpc GetBuild(GetBuildRequest) returns (stream GetBuildReply) {}
+  rpc GetDependencies(GetDependenciesRequest) returns (stream GetDependenciesReply) {}
```

Comment:
Wondering if this should this rather be a unary call instead of a server stream call?

It would make this code a little bit cleaner:

https://github.com/microsoft/vscode-gradle/blob/cc08a1fc02dcc8c2b5a480497622eebb19946838/extension/src/client/GradleClient.ts#L236-L242

For unary implementation on the server you can refer to:

https://github.com/microsoft/vscode-gradle/blob/cc08a1fc02dcc8c2b5a480497622eebb19946838/gradle-server/src/main/java/com/github/badsyntax/gradle/handlers/GetDaemonsStatusHandler.java#L29-L40

(Feel free to ignore me, i'm doing a quick "drive-by" review.)

Reply:
@badsyntax Thanks a lot for the review! The unary call works in this request and it's indeed cleaner. I'll update this PR later :)